src/sql/workbench/parts/connection/common/connectionProviderExtension.ts

```diff
@@ -165,6 +166,7 @@ ExtensionsRegistry.registerExtensionPoint<ConnectionProviderProperties | Connect
 
 	for (let extension of extensions) {
 		const { value } = extension;
+		//need to figure out when an extension is already registered so to avoid this process altogether.
```

Comment:
I think you can get rid of that comment all together too - what's happening here doesn't seem like a big deal. Unless @Charles-Gagnon you think we should fix that too?

Reply:
I understand, this is merely something to bypass the bug for now, instead of actually handling the issue, but at the moment there doesn't appear to be a easily visible means to see if the extension is registered or not.

Reply:
The extensionsRegistry has a way to see if an extension is registered or not.

src\vs\workbench\services\extensions\common\extensionsRegistry.ts

Reply:
That's fair but the bug doesn't really harm anything from what I've seen. It's just an error that's printed out, functionally everything else seems to work. So I'd rather we spend the time to fix this correctly rather than band-aid'ing it in the short-term.

Reply:
Thanks for the heads up @aaomidi I'll look at it and use it to resolve the issue properly

Reply:
As for how we handle this - what about tracking that here? If all registrations go through this code then we should be able to keep a map of registered extensions in this code as it is.